### PR TITLE
Added missing aliases

### PR DESCRIPTION
**Pull request by mathiascode**
This PR adds missing WorldEdit aliases to the plugin.

### CODE REVIEW

**Comment by mathiascode** [comment]
Don't merge yet. I forgot to update the readme.

File: Info.lua [hunk 1]
```diff
@@ -448,47 +481,50 @@ g_PluginInfo =
...
 			Handler = HandleBiomeListCommand,
-			HelpString = " Gets all biomes available",
+			HelpString = "Gets all biomes available",
 			Category = "Biome",
 		},
 		
 		["/brush"] =
 		{
-			Alias = { "//brush", "/br", "//br", },
+			Alias = "/br",
```
Comment (NiLSPACE) [inline Info.lua]: `//brush` and `//br` are not official aliases?

File: Info.lua [hunk 2]
```diff
@@ -581,72 +618,110 @@ g_PluginInfo =
...
 		["/toggleeditwand"] =
 		{
 			Permission = "worldedit.wand.toggle",
 			Handler = HandleToggleEditWandCommand,
-			HelpString = " Toggle functionality of the edit wand",
+			HelpString = "Toggle functionality of the edit wand",
 			Category = "Special",
 		},
 		
+		["/tool"] =
```
Comment (NiLSPACE) [inline Info.lua]: Were we missing this in the `Info.lua` file?

**Comment by NiLSPACE** [comment]
I assume these are all official aliases?

File: Info.lua (hunk 1 above)
Comment (mathiascode) [inline Info.lua]: Sorry, it seems like they are official after all.

File: Info.lua (hunk 2 above)
Comment (mathiascode) [inline Info.lua]: The /tool command was missing.

**Comment by mathiascode** [comment]
Yes, they are official.

**Pull request merged**